lab_1/main.py

```diff
@@ -116,4 +283,10 @@ def save_profile(profile: dict) -> int:
     :param profile: a dictionary
     :return: 0 if everything is ok, 1 if not
     """
-    pass
+    if not isinstance(profile, dict):
+        return 1
+    name = profile['name']
+
+    with open ("../lab_1/profiles/"+ name + ".json", "w", encoding = "utf-8") as new_file:
```

Comment:
It is ok for now. Yet I **highly** suggest reading about `os.path` and `Path` library. You can find an example in the `start.py`.